app/assets/stylesheets/quby/screen.css.scss

```diff
@@ -19,13 +19,18 @@ sup {
     top: -0.4em;
 }
 
+html {
+  overflow-x: hidden;
```

Comment:
Hierdoor kun je ook niet meer handmatig niet meer naar rechts scrollen. Dit is bijvoorbeeld een probleem met iPhone SE met `denk` vragenlijst.
![image](https://cloud.githubusercontent.com/assets/8157262/22016881/2135983a-dca8-11e6-81b7-b12b3390ee73.png)